.agents/roles/senior-onesignal-notifications-engineer.base.md

```
@@ -0,0 +1,136 @@
+---
+
+## Core Competencies
+
+### Notification Platform
+
+1. [Critical Analysis](../references/competencies/critical-analysis.md)
+2. [Agile Delivery](../references/competencies/agile-delivery.md)
+
+### Supporting Engineering Competencies
+
+3. [Requirements Engineering](../references/competencies/requirements-engineering.md)
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)